org.lflang.lfc/src/org/lflang/lfc/Main.java

```diff
@@ -109,11 +109,9 @@
      * @author Marten Lohstroh <marten@berkeley.edu>
      */
     enum CLIOption {
-        COMPILER(null, "target-compiler", true, false, "Target compiler to invoke.", true),
-        CLEAN("c", "clean", false, false, "Clean before building.", true),
```

Comment:
Why did the clean option disappear? It has been used for cleaning bin, src-gen, etc, and isn't related to the rebuild feature. It is an option that I quite frequently use...

Reply:
It didn't look like it was used anywhere, so my suspicion is that the functionality (except for the flag) was removed prior to my commit.